### ISSUE TITLE
Process birthdate from exit survey form before download

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#410]
**TL;DR**: When researchers download response data, the birthdate field from any exit-survey frame should be replaced with a field that just says how many days it's off from the birthdate of the registered participant. 

**Narrative**
"As an Experimenter, I use the birthdate field from the exit survey to double-check participant ages, but I don't want to publish birthdates (or even download them if I can avoid it) to protect participant privacy."

**Acceptance Criteria**
- [x] JSON and frame data downloads on "all responses" view have DOB difference (form birthdate - registered birthdate, in days) rather than DOB from exit survey
- [x] JSON and frame data downloads on "individual responses" view have DOB difference rather than DOB from exit survey
- [x] Tests for birthdate redaction added: check that there is no birthdate field for an EXIT type frame, but that there is for a frame with another type if it's included in the data; check that the difference is as expected for a few cases (+, -, 0)
